### ISSUE TITLE
Changelog out of date since v4.2.1

### ISSUE THREAD

**Issue by dplanella** [gollum/gollum-lib#265]
I've just noticed that there is an update for gollum-lib available on my system (`gollum-lib (4.2.2 < 4.2.5)`
). Yet by looking at the changelog in [HISTORY.md](https://github.com/gollum/gollum-lib/blob/master/HISTORY.md) I can't really tell which changes have been made in this new version, as the file was not updated since version 4.2.1.

How are releases made? I would assume that there is some degree of scripting/automation at least to create the gems. Could the script be adapted to automatically generate the changelog/bump versions, so that it is not a manual process and it is not forgotten about?

It'd be nice to list the bugs that have been fixed from release to release too. Happy to try contributing if I get some pointers.

Keep up the good work!

**Comment by dometto** [comment]
Yes, we release using the `Rakefile` (with `bundle exec rake release`, rake being 'ruby make'). Minimally it could throw a warning or error if `History.md` is not updated -- that would be trivial to implement. Ideally, there would be some degree of automation of updating the changes based e.g. on commit messages. But it would be painful to implement that ourselves. If you could look around and see whether there are any standardised ruby/rake release tools that do this, that would be great!

**Comment by dometto** [comment]
Updated for 5.0.

**Comment by dometto** [comment]
Implementing a `rake` helper task for Gollum, see here: https://github.com/gollum/gollum/pull/1788. We can copy it to this repo.

**Comment by dometto** [comment]
Done in https://github.com/gollum/gollum-lib/pull/421
